### ISSUE TITLE
0crat should open issues in its own repo if something fails

### ISSUE THREAD

**Issue by amihaiemil** [zerocracy/farm#500]
Currently, if there is an exception, 0crat anounces the commander and asks them to subit an issue, also printing the exception. 

I think it would be more elegant if 0crat cauth all the exception at the highest level and open an issue in its repo itself, then letting the commander know that something went wrong and an issue has been opened about it. 

For instance, one of my bots performs Actions (the Action is the highest abstraction), like this:

```java
new VigilantAction(
    new Chat(
        github.repos().get(
            new Coordinates.Simple(
                notification.repoFullName()
             )
        ).issues().get(notification.issueNumber()),
        new GithubSocialSteps()
    ),
    github
)
```

Chat is the original action, and [VigilantAction](https://github.com/amihaiemil/comdor/blob/d0bb2d56215ee975372925767bf65d1e8967fc6f/src/main/java/co/comdor/VigilantAction.java#L65) catches any exceptions from it and submits an Issue. Could something similar be implemented in 0crat?

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by yegor256** [comment]
@amihaiemil well... not a bad idea. Currently we are submitting all issues to Sentry and then from there we can create GitHub issues, but manually. I think that if we do it automatically, GitHub repo will be flooded with issues (some errors repeat very frequently). How would we filter our duplicates?

**Comment by amihaiemil** [comment]
@yegor256 Yes, there isn't much to do about duplicates... But how often can a problem repeat? If it's some communication issue with AWS that should be fixed quickly. Sure, some HTTP calls may fail every now and then but that isn't supposed to happen often. 

Maybe implement this after we have a more stable version that won't flood the repo with issues?

**Comment by amihaiemil** [comment]
@yegor256 Or we can implement it and also add a switch on the UI somewhere (if there is an admin panel).  Then, we can activate/deactivate the functionality whenever we want :-?

**Comment by yegor256** [comment]
@amihaiemil let's keep it open. Maybe in the future something like this will make sense.
